src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/utils/TaskMapper.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.utils;
```

Comment:
We need to restore the previous deleted class and place this one in:
package uk.gov.hmcts.reform.wataskmanagementapi.services;
Since this is a service, not util class :)